lib/secrets.rb

```diff
@@ -48,6 +50,11 @@ def initialize(
     @secret_keys = secrets_from_annotations(annotations)
     raise "#{annotations} contains no secrets" if @secret_keys.empty?
     @logger.info(message: "secrets found", keys: @secret_keys)
+
+    @pki_keys = pki_from_annotations(annotations)
+    unless @pki_keys.empty?
+      @logger.info(message: "PKI found", keys: @pki_keys)
+    end
```

Comment:
maybe always log so debugging is easier when non were found ?
```suggestion
      @logger.info(message: "PKIs", keys: @pki_keys)
```